pkg/api/pss.go

```diff
@@ -86,6 +82,20 @@ func (s *server) pssPostHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *server) pssWsHandler(w http.ResponseWriter, r *http.Request) {
+	upgrader := websocket.Upgrader{
+		ReadBufferSize:  swarm.ChunkSize,
+		WriteBufferSize: swarm.ChunkSize,
+		CheckOrigin: func(r *http.Request) bool {
+			o := r.Header.Get("Origin")
+			if o != "" {
+				return true
+			}
+			if o != "" && (s.CORSAllowedOrigins == nil || containsOrigin(o, s.CORSAllowedOrigins)) {
+				return true
```

Comment:
this condition will never get hit, because you will always return anyway on the previous `if` statement, as it comes before.

Reply:
omg, yep, fixed, i am having one of those mornings, fml 🙈

Reply:
i wonder if `o == "" || o != ""` makes sense here. maybe `if s.CORSAllowedOrigins == nil || containsOrigin(o,s.CORS...)` is enough. can you confirm that this works? I guess it would be nice to have a unit test on this, but manual testing due to lack of time will do at this point